internal/util/util.go

```diff
@@ -144,6 +145,21 @@ func GetSecretPassword(clientset kubernetes.Interface, db, suffix, Namespace str
 
 }
 
+// GetSidecarImageTag takes the current image name and the image tag value
+// stored in the pgcluster CRD and, if the image being used is the
+// crunchy-postgres-gis-ha container with the corresponding tag, it returns
+// the tag without the addition of the GIS version. This tag value can then
+// be used when provisioning sidecars using the standard containers tag.
+func GetSidecarImageTag(imageName, imageTag string) string {
+
+	if imageName == "crunchy-postgres-gis-ha" && strings.Count(imageTag, "-") == 3 {
+		re := regexp.MustCompile(`(.+-.+)-.+(-.+)`)
+		return re.ReplaceAllString(imageTag, "$1$2")
+	}
```

Comment:
Seems like this will be problematic when testing RC's, and we therefore have an extra `-` in our tags, e.g.:

```
centos8-13.0-3.0-4.5.0-rc.1
```

...would simply return the same value as passed in.

Reply:
Good catch, thank you @andrewlecuyer. I'll work on some additional tests to catch that and any other cases that may come up.